### DISCUSSION TITLE
"Docs" Not Displayed in Header on Homepage

### DISCUSSION THREAD

**Discussion by ghost** [colinwilson/lotusdocs#48] [Q&A]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for support

### Describe the bug

Based on [this](https://lotusdocs.dev/docs/quickstart/) quickstart guide, I created a page in the "docs" section. 

The URL `http://127.0.0.1:1313/docs` works and the page is listed, but the "docs" section is not visible in the homepage `http://127.0.0.1:1313`. 

I'm unable to find information on [lotusdocs.dev](https://lotusdocs.dev/) about configuring the header.

### Expected behavior

Docs to be displayed in the header on the homepage.

### Environment

```markdown
- OS:
- Browser:
- Browser Version:
```

**Comment by colinwilson** [comment]
> I'm unable to find information on [lotusdocs.dev](https://lotusdocs.dev/) about configuring the header.

Yep. I haven't documented configuring menus just yet (I will soon though). Will update here once I do.

**Comment by ghost** [comment]
This is such a beautiful theme! Thank you!

**Comment by ghost** [comment]
> Yep. I haven't documented configuring menus just yet (I will soon though). Will update here once I do.

Is there a way to create a PR for this?

**Comment by colinwilson** [comment]
There isn't just yet.

But only because many sections of the documentation are in flux at the moment with whole topics missing and other sections likely to be removed, renamed or re-ordered. Hopefully won't be long, as I plan to finalise the core documentation by the end of Aug 2023 🙏.

Once the first release is available, PR's will be open (and welcomed) for the documentation site.

However, please do comment on anything you think should be documented. I appreciate all feedback and suggestions.
